frontend/public/ui/ui-reducers.js

```diff
@@ -89,6 +89,9 @@ export default (state, action) => {
     case types.selectOverviewDetailsTab:
       return state.setIn(['overview', 'selectedDetailsTab'], action.tab);
 
+    case types.dismissOverviewDetails:
+      return state.mergeIn(['overview'], {selectedUID: '', selectedDetailsTab: ''});
```

Comment:
Are we sure that we want to dismiss the selection when navigating away and coming back? Personally I like that it remembers.

(I agree the that tab should reset when closing the details panel, though.)

@openshift/team-ux-review

Reply:
@spadgett +1 that the selection should be remembered.

Do we remember tab selection in any other areas of the UI?

Reply:
For tabs, we generally put the selected tab in the URL. When you use the browser back button, it's still selected, but if you navigate back later through the nav, it will be reset.

Current overview behavior is a little different since it's stored in Redux. It's always remembered here no matter how you navigate to the page.

We could look at adding it to the URL. It'd let you send it as a link to someone else or bookmark it.

Reply:
Hmm, okay. I can fix this. I assumed that leaving the page and returning, we would want to reset the page to its initial state. I thought I had read that in the design spec somewhere, but I I think there are only specifications for the tab behavior within the details pane.